docs/source/tutorial.rst

```diff
@@ -169,5 +169,38 @@ under **Contents**
 CNMF (Constrained Non-Negative Matrix Factorization)
 =====================================================
 Now that we have some outputs for motion correction, we can run CNMF on these outputs to extract
-spatial and temporal components of neuron activity. 
+spatial and temporal components of neuron activity.
+
+Generally, nonrigid motion correction will perform better than rigid motion correction,
+so we will use the output from **opt1** to run CNMF. There are two ways to select this output
+as the input movie for the next item. The first way will require that you select the mcorr item,
+go to the parameter viewer, and find the uuid.
```

Comment:
No, users should never navigate the batch dir structure themselves

Reply:
They do this to get the initial input movie though, no? For simplicity's sake, wouldn't it make sense to give an option on how to navigate directory through the same way they find input movie as well?

Reply:
I get your point actually, uuid is a pain to follow. Would be better to just use the recent input movie path. Sure we should just omit how to manually search? Might be helpful for when they're using notebooks to know how the batch dir structure works?

Reply:
They will never navigate the batch dir structure themselves, that's what the pandas extensions are for.